tests/conftest.py

```
@@ -763,42 +763,80 @@ def enable_stdout_viewer(self) -> None:
+    ]
+
+    import sys
+
+    for mod_name in modules_with_telemetry:
+        try:
+            if mod_name in sys.modules:
+                mod = sys.modules[mod_name]
+                if hasattr(mod, "telemetry"):
+                    monkeypatch.setattr(mod, "telemetry", mock_telemetry)
+        except Exception:
+            pass
```

Ruff (0.14.7)

`try`-`except`-`pass` detected, consider logging the exception

(S110)